### PR TITLE
German translation of the setting tables and numerous corrections

### PR DESCRIPTION
**Pull request by Mikelectric**
In addition to the new settings, I also corrected many old translations that often caused misunderstandings. Adjustments to the new German spelling. Correction of spelling mistakes. Everything checked and tested.

Some translations of the .po file are obviously not used in the program at all.

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -548,6 +563,12 @@ msgstr "Audio-Gruppe %d"
 msgid "Audio group name"
 msgstr "Neuer Audio-Gruppenname"
 
+msgid "Audio recording:"
+msgstr "Audio-Aufnahme:"
+
+msgid "Auto add new devices"
+msgstr "Neue Geräte automatisch hinzufügen"
```
Comment (oleg68) [inline po/de.po:570]: It seems too long. Can you make it shorter?

Comment (Mikelectric) [reply po/de.po:570]: It actually still fits well with the smallest window width.

Bildschirmfoto von 2022-01-26 20-15-59

**Review by oleg68 — APPROVED** [review]

**Comment by oleg68** [comment]
Thank you very much, @Mikelectric , for your contributing.

**Comment by oleg68** [comment]
@larspalo @rousseldenis could you approve this PR?

**Review by larspalo — APPROVED** [review]

**Comment by Mikelectric** [comment]
Thank you @oleg68 and @larspalo. Glad I was able to contribute and help.

**Pull request merged**